atlp-rwanda/atlp-pulse-bn
# Send email to trainee

### PR Description
fix redirect URL send to email
### Description of tasks that were expected to be completed
In this PR  I changed the way redirect URL is visible on email before it was like batch of text link included token but now you click on join us and redirect to our front-end registration part
### How has this been tested?

- Login as admin
- Invite user which is not contain in our organization
- check your email when you will get new email for devpulse

### PR Checklist:
- [ ] Task 3.
- [ ] Task n.
### Track PR
Trello Link (#DP-?)
### Screenshots (If appropriate)
N/A
